vaccine_feed_ingest/runners/la/tableau/normalize.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("la/tableau/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return f"{maybe_provider[0]}:{maybe_provider[1]}"
+
+    return site["name"]
```

Comment:
IDs shouldn't have spaces in them — check out [this PR](https://github.com/CAVaccineInventory/vaccine-feed-ingest/pull/186) to see how someone else normalized location names -> IDs